src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
+@use 'variables.scss' as *;
```

Comment:
Ensure that the variables `$stopwatch--size`, `$minute-hand--height`, `$minute-hand--width`, `$second-hand--height`, `$second-hand--width`, and `$stopwatch__dot--size` are defined in the 'variables.scss' file. If they are not defined, it will cause errors during the compilation of SCSS to CSS.